community/modules/file-system/nfs-server/outputs.tf

```diff
@@ -16,14 +16,14 @@
 # render the content for each folder
 output "network_storage" {
   description = "export of all desired folder directories"
-  value = [for mount in var.local_mounts : {
+  value = [for mount, mount_runner in local.mount_runners : {
     remote_mount          = "/exports${mount}"
     local_mount           = mount
-    fs_type               = "nfs"
-    mount_options         = "defaults,hard,intr"
-    server_ip             = google_compute_instance.compute_instance.network_interface[0].network_ip
+    fs_type               = local.fs_type
```

Comment:
nit: why the indirection? Do you expect it to change?

Reply:
It is also used [here](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/5ac4aab7417ed3cbadaf63d9ce0df82773996605/community/modules/file-system/nfs-server/main.tf#L35). So instead of using the same string literal twice I created a variable (kind of like a constant). Just my DRY coding habits kicking in. 

You could argue that the value from having the variable defined once is negated by diminished readability. But since you opened it as nit I will Ack and press forward.